src/structlog/_log_levels.py

```diff
@@ -91,16 +92,24 @@ def exception(
 async def aexception(
     self: FilteringBoundLogger, event: str, *args: Any, **kw: Any
 ) -> Any:
+    """
+    .. versionchanged:: 23.3.0
+       Implemented `contextvars.ContextVar` for holding and resetting async calling stack
+    """
     # Exception info has to be extracted this early, because it is no longer
     # available once control is passed to the executor.
     if kw.get("exc_info", True) is True:
         kw["exc_info"] = sys.exc_info()
 
+    _scs_token = async_calling_stack.set(sys._getframe().f_back)
```

Comment:
There's no reason to have an underscored variable in a function, I think?

Reply:
Truth -- cleaned up